inc/Nav_Menus/Component.php

```diff
@@ -59,15 +69,40 @@ public function template_tags() : array {
 		);
 	}
 
+	/**
+	 * Set the navs menus slugs and their description.
+	 *
+	 * @return void
+	 */
+	private function set_nav_menus() {
+		$this->nav_menus = array(
+			static::PRIMARY_NAV_MENU_SLUG => esc_html__( 'Primary', 'wp-rig' ),
+		);
+	}
+
+	/**
+	 * Get the navs menus slugs and their description.
+	 *
+	 * @return array Associative array of menu location identifiers (like a slug) and descriptive text.
+	 */
+	public function get_nav_menus() : array {
+		return $this->nav_menus;
+	}
+
+	/**
+	 * Get the navs menus slugs.
+	 *
+	 * @return array Array of menu location identifiers (like a slug).
+	 */
+	public function get_menu_slugs() : array {
```

Comment:
Revisiting this, I think this method isn't necessary. You could easily use `get_nav_menus()`. If you need to check for something using `isset()` is faster than `in_array()` anyway. :)